src/fft-api/common/fftApiClientService.ts

```diff
@@ -5,9 +5,9 @@ export class FftApiClient {
   private readonly baseUrl: string;
   private readonly authService: AuthService;
   private readonly httpClient: HttpClient;
-  constructor(projectId: string, username: string, password: string, apiKey: string) {
+  constructor(projectId: string, username: string, password: string, apiKey: string, logging: boolean | undefined) {
```

Comment:
Maybe the argument 'logging' should be renamed to something more specific like 'logHttpRequests' because
this flag is specific for the HttpClient and does not enable/disable logging globally for the FftApiClient

Reply:
see recent push, done.

Reply:
Done.